libkbfs/tlf_journal.go

```diff
@@ -105,6 +105,13 @@ const (
 	TLFJournalBackgroundWorkEnabled
 )
 
+type tlfJournalPauseType int
+
+const (
+	journalPausedFromConflict tlfJournalPauseType = 1 << iota
+	journalPausedFromSignal
```

Comment:
`FromCommand` seems to fit better, since with my suggestion below, all pauses will raise a signal on `needPause`.

Also maybe the format `journalPauseConflict` and `journalPauseCommand` is better, since the journal isn't necessarily paused yet once we raise a pause signal.